src/CommunityToolkit.Maui.Markup/ViewInAbsoluteLayoutExtensions.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Layouts;
+
+namespace CommunityToolkit.Maui.Markup;
+
+/// <summary>
+/// Absolute Layout Extension Methods
+/// </summary>
+public static class ViewInAbsoluteLayoutExtensions
+{
+	/// <summary>
+	/// Set LayoutFlags
+	/// </summary>
+	/// <typeparam name="TView"></typeparam>
+	/// <param name="view"></param>
+	/// <param name="flag"></param>
+	/// <returns></returns>
+	public static TView LayoutFlags<TView>(this TView view, AbsoluteLayoutFlags flag) where TView : View
+	{
+		AbsoluteLayout.SetLayoutFlags(view, flag);
```

Comment:
I can't tell if this is a crazy idea but should we throw an `InvalidOperationException` or similar if the parent isn't an `AbsoluteLayout`?

Reply:
Good point, in xaml if the parent isn't a absoluteLayout nothing happens. Just bring it out because I'm Not sure if we want to keep the same behavior or not

Reply:
`ViewInFlexLayoutExtensions` methods do not throw errors. Consider the situation where one Visual element that can be a child of an AbsouteLayout and a Grid  in runtime.

Reply:
Perhaps here we could offer an extension to `AbsoluteLayout` that accepts the flags and bounds?

Reply:
I agree with @danielftz. 

In Xamarin.Forms / .NET MAUI, It is possible to initialize a `View`, assign its LayoutBounds, then insert it into an AbsoluteLayout.

If we throw an `InvalidOperationException()` when the parent hasn't already been set, then we break this paradigm.